### ISSUE TITLE
Error incorrectly saying mutagen isn't installed

### ISSUE THREAD

**Issue by ShrinivasKR** [yt-dlp/yt-dlp#1809]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2021.11.10.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

I get an error saying that mutagen isn't installed although it has been installed using homebrew (same way I installed yt-dlp). mutagen does get recognized as a valid command when entered into the terminal by itself so it must be in the path.

### Verbose log

```shell
yt-dlp -f 'bestaudio' --download-archive downloaded.txt --continue --no-overwrites --ignore-errors --extract-audio --embed-thumbnail --add-metadata --cookies /Users/shrinivas/Downloads/cookies.txt PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ -Uv

[debug] Command-line config: ['-f', 'bestaudio', '--download-archive', 'downloaded.txt', '--continue', '--no-overwrites', '--ignore-errors', '--extract-audio', '--embed-thumbnail', '--add-metadata', '--cookies', '/Users/shrinivas/Downloads/cookies.txt', 'PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_', '-Uv']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1
[debug] Python version 3.9.9 (CPython 64bit) - macOS-12.0.1-x86_64-i386-64bit
[debug] exe versions: ffmpeg 4.4.1 (setts), ffprobe 4.4.1, rtmpdump 2.4
[debug] Optional libraries: sqlite
[debug] Proxy map: {}
[debug] Loading archive file 'downloaded.txt'
yt-dlp is up to date (2021.11.10.1)
[debug] [youtube:playlist] Extracting URL: PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_
[debug] [youtube:tab] Extracting URL: https://www.youtube.com/playlist?list=PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_: Downloading webpage
WARNING: [youtube:tab] YouTube said: INFO - Unavailable videos are hidden
[debug] [youtube:tab] Final URL: https://www.youtube.com/playlist?list=PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_
[debug] [youtube:tab] Extracted SAPISID cookie
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_: Downloading API JSON with unavailable videos
WARNING: [youtube:tab] YouTube said: INFO - Unavailable videos will be hidden during playback
[download] Downloading playlist: Fun Zouk songs
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 1: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 2: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 3: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 4: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 5: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 6: Downloading API JSON
[youtube:tab] PLCBPnj3xS_K4UmOJUIKHoK5eu51gS7Nr_ page 7: Downloading API JSON
[debug] Skipping writing playlist thumbnail
[youtube:tab] playlist Fun Zouk songs: Downloading 787 videos
[download] Downloading video 1 of 787
[debug] [youtube] Extracting URL: https://www.youtube.com/watch?v=Tf7mQl8TQe4
[youtube] Tf7mQl8TQe4: Downloading webpage
[debug] [youtube] Extracted SAPISID cookie
[youtube] Tf7mQl8TQe4: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, codec:vp9.2, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] Tf7mQl8TQe4: Downloading 1 format(s): 251
[info] Downloading video thumbnail 41 ...
[info] Writing video thumbnail 41 to: bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webp
[debug] Invoking downloader on "https://r3---sn-nx5s7n7y.googlevideo.com/videoplayback?expire=1638003683&ei=g5-hYaKwG5KYsfIP0oCHmAI&ip=2601%3A600%3Aa27f%3Ac0e0%3Ac1b0%3A3160%3A3a32%3A44d0&id=o-AJPrXzzpw7f0S5sRAuAtk-PtUpBYogcaEPe9KhUgIKBi&itag=251&source=youtube&requiressl=yes&mh=nk&mm=31%2C29&mn=sn-nx5s7n7y%2Csn-nx57ynlk&ms=au%2Crdu&mv=m&mvi=3&pl=35&ctier=A&pfa=5&initcwndbps=2122500&hightc=yes&vprv=1&mime=audio%2Fwebm&gir=yes&clen=2526918&dur=176.581&lmt=1627212530063513&mt=1637981821&fvip=3&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=5311224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cctier%2Cpfa%2Chightc%2Cvprv%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAKPmNbLu6h25_OJ20uDcCyoxU_tR9LUKD5hs-m7cK_7IAiEAm-WzTjR2AWTUExvbCM6YielBP_X4svE1IodczmKj8eo%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRgIhAOzVQokmLPuWenTHxDU8QFYVWznORJe2-OydaPlxT8y0AiEA5yyfCLAbNdI0wtTsg5fvSswi8_aeAxRYbW3npGSUJkE%3D"
[download] Destination: bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webm
[download] 100% of 2.41MiB in 00:00
[debug] ffmpeg command line: ffprobe -show_streams 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webm'
[ExtractAudio] Destination: bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].opus
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webm' -vn -acodec copy 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].opus'
Deleting original file bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webm (pass -k to keep)
[Metadata] Adding metadata to "bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].opus"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].opus' -map 0 -dn -c copy -metadata 'title=bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞' -metadata date=20210722 -metadata 'description=bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞
SUBSCRIBE, I WOULD APPRECIATE IT.
✖️ Channel is not monetized, if ads appear they are from copyright claims.

DONT FORGET TO SUBSCRIBE
Best place to relax/study to!
(chill beats, lofi hiphop, lofi covers)

🌎 Follow SheepVibes
→ instagram https://bit.ly/36ldUoK
→ twitter https://bit.ly/38tdIX3

Original Audio: https://www.youtube.com/watch?v=K6BRna4_bmg&ab_channel=bbno%24

~DISCLAIMER~
I do not own these gifs or songs. Full credit goes to the creators. If you are the creator of any of these and wish for me to remove it,  message me and I will take it out immediatly

~ TAGS ~

#Reverb #slowed #Music' -metadata 'synopsis=bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞
SUBSCRIBE, I WOULD APPRECIATE IT.
✖️ Channel is not monetized, if ads appear they are from copyright claims.

DONT FORGET TO SUBSCRIBE
Best place to relax/study to!
(chill beats, lofi hiphop, lofi covers)

🌎 Follow SheepVibes
→ instagram https://bit.ly/36ldUoK
→ twitter https://bit.ly/38tdIX3

Original Audio: https://www.youtube.com/watch?v=K6BRna4_bmg&ab_channel=bbno%24

~DISCLAIMER~
I do not own these gifs or songs. Full credit goes to the creators. If you are the creator of any of these and wish for me to remove it,  message me and I will take it out immediatly

~ TAGS ~

#Reverb #slowed #Music' -metadata 'purl=https://www.youtube.com/watch?v=Tf7mQl8TQe4' -metadata 'comment=https://www.youtube.com/watch?v=Tf7mQl8TQe4' -metadata artist=SheepVibes 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].temp.opus'
[ThumbnailsConvertor] Converting thumbnail "bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webp" to png
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -f image2 -pattern_type none -i 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].webp' 'file:bbno$ & Rich Brian - Edamame﹝slowed + reverb﹞ [Tf7mQl8TQe4].png'
ERROR: module mutagen was not found. Please install using `python -m pip install mutagen`
Traceback (most recent call last):
  File "/usr/local/Cellar/yt-dlp/2021.11.10.1/libexec/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 3022, in run_pp
    files_to_delete, infodict = pp.run(infodict)
  File "/usr/local/Cellar/yt-dlp/2021.11.10.1/libexec/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 22, in run
    ret = func(self, info, *args, **kwargs)
  File "/usr/local/Cellar/yt-dlp/2021.11.10.1/libexec/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 112, in wrapper
    return func(self, info)
  File "/usr/local/Cellar/yt-dlp/2021.11.10.1/libexec/lib/python3.9/site-packages/yt_dlp/postprocessor/embedthumbnail.py", line 199, in run
    raise EmbedThumbnailPPError('module mutagen was not found. Please install using `python -m pip install mutagen`')
yt_dlp.postprocessor.embedthumbnail.EmbedThumbnailPPError: module mutagen was not found. Please install using `python -m pip install mutagen`
```

**Comment by pukkandan** [comment]
> [debug] Optional libraries: sqlite

This should show mutagen if it was installed. I don't know how brew works, but I assume it was installed into a separate virtual environment

**Comment by Zirro** [comment]
Brew uses python@3.10 for recent yt-dlp versions, but it's still at python@3.9 for you suggesting it's been upgraded without using `brew`. Try running `brew remove yt-dlp && brew install yt-dlp` and see if that fixes it or outputs an error which you can share here.

**Comment by ShrinivasKR** [comment]
> Brew uses python@3.10 for recent yt-dlp versions, but it's still at python@3.9 for you suggesting it's been upgraded without using `brew`. Try running `brew remove yt-dlp && brew install yt-dlp` and see if that fixes it or outputs an error which you can share here.

AFAIK I only upgraded yt-dlp using brew. I first noticed this issue I think after I upgraded to MacOS Monterey. I wasn't getting the error prior to that. Seems pretty strange, but I don't know enough about how this stuff works to have any theories.

I had multiple times tried to remove and install mutagen to no avail.

I did however follow your suggestion to remove and install yt-dlp again using brew and am now no longer seeing the error. Thank you! 

I hope this information helps anyone else resolve the issue if they face it and the maintainers of the brew recipe/yt-dlp to make any changes if necessary.
